v2/thirdpartyemailpassword/nestjs/guide.mdx

```diff
@@ -36,16 +36,16 @@ You can scaffold a module using the nest CLI by running this in the root folder
 ```bash
 nest g module auth
 ```
-The result should be a new `auth` folder with `auth.module.ts` in it. We should convert this into a [dynamic module](https://docs.nestjs.com/modules#dynamic-modules) so we can set parts of the SuperTokens configuration in the App module. Centralizing settings like this can be helpful for things like using a separate connection URI for testing.
+The result should be a new `auth` folder with `auth.module.ts` in it. 
 
 ### Add config type and injection token
 
-Add a `config.interface.ts` file next into the `auth` folder. We will put the type and injection token for the SuperTokens config here.
+Create a `config.interface.ts` file in the `auth` folder. We will put the type and injection token for the SuperTokens configuration here.
 
-```ts
-import { AppInfo } from "supertokens-node/types";
+```ts title="./src/config.interface.ts"
```

Comment:
this is not the rigth path, since the line above says in the auth folder, but the title of this snippet says src/config..

Reply:
I changed this file to be located in `./src/auth`.